assets/base.css

```diff
@@ -2759,7 +2853,7 @@ details-disclosure > details {
   font-size: 1.2rem;
   letter-spacing: 0.1rem;
   line-height: 1;
-  padding: 0.5rem 1.3rem 0.6rem 1.3rem;
+  padding: 0.5rem 1.3rem 0.6rem;
```

Comment:
Here I kept the original approach as when the value is the same on the right and left the use of logical properties isn't necessary. 
But open to change it to have everything use logical properties. I'll wait for dev feedback on this.

Reply:
Maybe for consistency we should use the new approach. If in the future left and right are not the same we won't need to fix them.